src/Xamarin.Android.Build.Tasks/Resources/LayoutBinding.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+
+using Android.App;
+using Android.Views;
+
+namespace Xamarin.Android.Design
+{
+	abstract class LayoutBinding
+	{
+		public delegate Java.Lang.Object OnLayoutItemNotFoundHandler (int resourceId, Type expectedViewType);
```

Comment:
Should this be an explicit nested type? Or should it instead be `Func<int, System.Type, Java.Lang.Object>`? `Func` would be shorter (entire reason I bring it up is that `global::Xamarin.Android.Design.LayoutBinding.OnLayoutItemNotFoundHandler` is "long" to read & mentally process).